src/keys/CompositeKey.cpp

```diff
@@ -64,10 +67,14 @@ CompositeKey& CompositeKey::operator=(const CompositeKey& key)
 
     clear();
 
-    for (const Key* subKey : asConst(key.m_keys)) {
+    Q_FOREACH (const Key* subKey, key.m_keys) {
```

Comment:
Duplicate of the previous PR: https://github.com/keepassxreboot/keepassxc/pull/119#pullrequestreview-11167761